eXoUITests/TestSetting.swift

```diff
@@ -64,7 +65,8 @@ class TestSetting: eXoUIBaseTestCase {
         }
     }
 
-    
+//KMenzli: Test fail on Travis CI when activated
```

Comment:
Can you replace your name by the jira issue MOB-2054 please ? (and indent it correctly)